### DISCUSSION TITLE
Build Polynesian models and language

### DISCUSSION THREAD

**Discussion by MauiDesign** [explosion/spaCy#11876] [Language Support]
hi,
Looking to build Polynesian languages for spaCy. 

Would like to know the steps to kick start the build.

Thanks

**Comment by ljvmiranda921** [comment]
Hi @MauiDesign ,

You can find some of the initial steps in this [discussion](https://github.com/explosion/spaCy/discussions/3056) for adding new languages or building language-specific models in spaCy. Most likely, your first step would be to find an annotated corpus.  Lastly, I highly recommend checking out the ["new languages" Issue label](https://github.com/explosion/spaCy/discussions?discussions_q=label%3A%22new+language%22) every now and then to see the usual process and challenges.

**Reply by MauiDesign** [reply]
thank you
